### ISSUE TITLE
Why no table style

### ISSUE THREAD

**Issue by narukeh** [susam/spcss#6]
a table without a simple style looks really dumb. So why not have at least sth as simple as this:

```css
table, th, td {
  border: 1px solid #aaa;
  border-collapse: collapse;
  overflow: auto;
  padding: 2px 5px;
}
```
![pic-selected-201227-0300-07](https://user-images.githubusercontent.com/62762955/103162357-80de4580-47e7-11eb-9d50-9e50880d3ec7.png)

-----
(And btw, do you happen to know why firefox does not respect the theme i chose? Its not just with your `index.html` , its with all. I have chosen the dark theme, though websites still offer me the light verions)

**Comment by susam** [comment]
Thank you for the suggestion. I have added a style for table now in commit ff15568. A copy of it is available at https://susam.github.io/spcss/sp.css. It will be available on the CDNs on the next release (sometime next week).

I don't know what issue you might be facing with Firefox. I enable dark theme at the operating system level (Apple menu > System Preferences > General > Appearance, on macOS Catalina 10.15.7). This works for me. If an operating system does not have a setting like this, then the MDN Web Docs has a workaround [here](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) that too works for me. The workaround involves going to `about:config`, then creating a new preference named `ui.systemUsesDarkTheme` of numeric type, and setting its value to `1`. I hope that helps.

**Comment by narukeh** [comment]
Thanks for replying. Great news. Also thanks for the siggestion, i will try that out.

**Comment by susam** [comment]
A minimal style for the `<table>` element is now available in [version 0.6.0](https://www.jsdelivr.com/package/npm/spcss?version=0.6.0). See the [changelog](https://github.com/susam/spcss/blob/master/CHANGES.md) for more details.
